content/docs/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Docs Index"
+title: "Overview"
```

Comment:
Comment: It would be good to provide a _general_ blurb on "what is Dex?" on this page.

cc @justaugustus, please tag other maintainers as you see fit.

Reply:
Comment #2: 

Please work with the maintainers to decide on the correct order for pages in these sections.

Reply:
I agree with both these comments, and would like to break them into separate issues to be done after this initial PR.